crane/container.go

```diff
@@ -2,13 +2,14 @@ package crane
 
 import (
 	"fmt"
-	"github.com/flynn/go-shlex"
-	"github.com/michaelsauter/crane/print"
 	"io"
 	"os"
 	"path"
 	"strconv"
 	"strings"
+
+	"github.com/flynn/go-shlex"
+	"github.com/michaelsauter/crane/print"
```

Comment:
Interesting. Anyway, the pull request now removes `go-shlex` which is definitely needed, so this cannot compile. Please fix (and ideally I'd like the import statements not to change at all, since they're not related to this PR).